Gruntfile.js

```diff
@@ -243,7 +257,11 @@ module.exports = function (grunt) {
   });
 
   grunt.registerTask('default', ['newer:babel', 'less', 'newer:copy:dev', 'shell:electron', 'watchChokidar']);
-  grunt.registerTask('release', ['clean:release', 'babel', 'less', 'copy:dev', 'electron', 'copy:osx', 'shell:sign', 'shell:zip', 'copy:windows', 'rcedit:exes', 'compress']);
+  if(process.platform === 'linux') {
```

Comment:
Oh ! Sure ! Sorry, I had some difficulties when I did the `rebase`. I'll fix it.